discovery/amazon/lightsail.go

```diff
@@ -0,0 +1,235 @@
+// Copyright 2015 The Prometheus Authors
```

Comment:
```suggestion
// Copyright 20121The Prometheus Authors
```

Reply:
Updated copyright in `ec2.go` & `lightsail.go`